gems/pending/appliance_console.rb

```diff
@@ -409,8 +421,11 @@ module ApplianceConsole
         if agree("Setting Database Region to: #{region_number}\nAre you sure you want to continue? (Y/N): ")
           say("Setting Database Region...  This process may take a few minutes.\n\n")
 
-          if Env.rake("evm:db:region -- --region #{region_number} 1>> #{LOGFILE}")
-            say("Database region setup complete...\nStart the #{I18n.t("product.name")} server processes via '#{I18n.t("advanced_settings.evmstart")}'.")
+          if ApplianceConsole::Utilities.rake("evm:db:region", "--", "--region", region_number.to_s)
```

Comment:
Won't this always be true, even in the failure case, because you will return the exit code?